static/app/views/pipeline/coverage/commits/commitDetail.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import CommitDetailPage from 'sentry/views/pipeline/coverage/commits/commitDetail';
+
+const COVERAGE_FEATURE = 'codecov-ui';
+
+describe('CommitDetailPage', () => {
+  it('renders the passed children', () => {
```

Comment:
Nit: technically not passing children here but that might change later since placeholder so fine with your choice 👍

Reply:
whoops left over from copy paste